README.md

```diff
@@ -28,6 +28,14 @@ The DevOps working group is an umbrella for multiple, more specific projects. Se
 - Communication: [Developer Experience](https://discuss.openedx.org/c/working-groups/dev-experience) working group forum category and #wg-developer-experience Slack channel.
 - Issue management: [Developer Experience working group](https://github.com/orgs/openedx/projects/37)
 
+### Large Instances working group
+
+- General information: [Large Instances workspace](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3655008783/Large+Instances+-+DevOps+Sub-Working+Group)
+- Communication: [bi-weekly meeting](https://discuss.openedx.org/t/deploying-open-edx-on-kubernetes-using-helm/8771), [DevOps](https://discuss.openedx.org/c/operators/7) forum category, [#wg-devops](https://openedx.slack.com/channels/wg-devops) Slack channel.
+- Issue management: [Large Instances working group](https://github.com/orgs/openedx/projects/42/views/2)
+
+This group leads the Harmony Kubernetes project, used to deploy Open edX via Helm & Tutor: https://github.com/openedx/tutor-contrib-multi
```

Comment:
Can you replace by the new project name?

Reply:
@regisb Sure - since the name was not yet changed when I opened this PR, I had kept the original repo URL (which would then be redirected to the new name), but now that the name change is done, you're right that it's worth using the final URL:

```suggestion
This group leads the Harmony Kubernetes project, used to deploy Open edX via Helm & Tutor: https://github.com/openedx/openedx-k8s-harmony
```